### DISCUSSION TITLE
Possible to get all Tables inside of the Database ? (MySQL, Next.JS)

### DISCUSSION THREAD

**Discussion by Gipfel** [prisma/prisma#12920] [Q&A] [answered]
Hey there,
I am just searching if there is any possibility with the help of Prisma.JS to get all Tables in the Database, but I haven't found anything.
Can anyone please help me out there?

I need that because I am working on a Next.JS Project where every section is built up on Database data on his own table.
I already got that many that the Application isn't stable enough any more. So my Idea was to just grab every Table with its data and give it via Props to the Components.

Thank you in advance!!!

**Comment by Gipfel** [comment, marked as answer]
I have found a workaround!
```
async function prismaFetched() {
  const matches = await prisma.matches.findMany()
  const partners = await prisma.partners.findMany()
  const shop = await prisma.shop.findMany()
  const socialmedia = await prisma.socialmedia.findMany()

  return [matches, partners, shop, socialmedia];
}
```
I am just doing the queries like normal, but I add it all to 1 Array and I give the information away by a Next.JS Api Route

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
